packages/playground/docs/getting-started-nested.md

```diff
@@ -7,32 +7,92 @@ permalink: /playground/
 
 # Getting Started
 
-The UI5 Web Components carry fundamental UI5 qualities, along with others, to provide enterprise-grade features, Fiori UX, and themeability. The goal is to achieve easy consumption of UI5 controls and to lower the entry barrier to use UI5 controls for own applications. By making the consumption of the full-stack UI5 optional, UI5 Web Components allow you to avoid the complexity of the UI5 rendering stack. In general, the UI5 Web Components are targeting web developers who want to have more flexibility to use just HTML tags or arbitrary JS frameworks.
+The UI5 Web Components share fundamental UI5 qualities with others to provide enterprise-grade features, Fiori UX and themeability. The goal is to achieve an easy consumption of UI5 controls to lower the entry barrier to use UI5 controls for own applications and to avoid complexity of the UI5 rendering-stack by making the consumption of the full-stack UI5 optional. In general, the UI5 Web Components target web developers who want to have more flexibility to use just HTML tags or arbitrary JS frameworks.
+
+UI5 Web Components are shipped as ES6 modules and published on NPM. If you are already using a framework that handles bundling of ES6 modules, skip forward to [Installation](#installation) or check the Tutorials section.
+
+## Creating a project
+
+In case you don't have an existing project with frontend tooling, you can start a new one using [Vite](https://vite.dev). The only prerequisite is to have [Node.js](https://nodejs.org/) installed.
+
+```console
+npm init @vitejs/app
+```
+
+- Select a project name: `my-webcomponents-app`
+- Select a framework: `vanilla`
+- Select a variant: `JavaScript`
+
+This command will create a project structure under `my-webcomponents-app`. Follow the instructions on the screen to install the initial depenednecies and start a dev server.
+
+```
+cd my-webcomponents-app
+npm install
+npm run dev​
+```
+
+You should see a message that a server is running on `Local: http://localhost:3000/`. Launch your favourite IDE and change some code, the browser page will be automatically refreshed.
 
 ## Installation
+<a name="installation"></a>
 
-### Install ```@ui5/webcomponents```
+Install ```@ui5/webcomponents```
 
 ```bash
-npm install @ui5/webcomponents
+npm install @ui5/webcomponents --save
 ```
 
-## Read the Docs
+This command downloads the published source code of UI5 web components to the `node_modules` folder. Now you can add them to your code to get them on the web page.
+
+## Documentation
 
-Documentation and tutorials about UI5 Web Components can be found [here](./playground/docs/)
+Browse the available UI5 Web Components and check their API in the [playground](./playground/docs/).
```

Comment:
Take a look at the available UI5 Web Components documentation

Reply:
done